content/game_process.tex

```diff
@@ -120,13 +118,23 @@ \subsubsection{Indirect Kick}
 \end{itemize}
 
 \begin{description}
-  \item[Example 1:] Player 2 (of the red team) kicks the ball to Player~3, who then kicks the ball into the goal. This is a successful indirect kick, and the goal counts.
-  \item[Example 2:] Player 2 (of the red team) kicks the ball at the goal, and it is deflected of the side of the foot of a blue-team robot into the goal. This is \textit{not} an indirect kick, and the goal does not count.
-  \item[Example 3:] Player 2 (of the red team) kicks the ball ``upfield''. A blue-team robot kicks the ball a short distance, after which Player~2 kicks the ball again into the goal. This is a successful indirect kick, and the goal counts.
-  \item[Example 4:] Player 2 (of the red team), walks up to and dribbles the ball. To be an indirect kick, Player~2 must then stop, and visibly back-away from the ball, before approaching to dribble a \textit{second} time. The robot then scores. This is a successful indirect kick.
-  \item[Example 5:] Players of the red team fail to execute a Kick off. After the ball is free the defending team kicks the ball directly into the goal. This is \textit{not} an indirect kick, and the goal does not count.
+  \item[Example 1:] Player 2 (of the red team) kicks the ball to Player~3, who then kicks the ball into the blue-team's goal. This is a successful indirect kick, and the goal counts.
+  \item[Example 2:] Player 2 (of the red team) kicks the ball at the goal, and it is deflected of the side of the foot of a blue-team's robot into the goal. This is \textit{not} an indirect kick, and the goal does not count.
+  \item[Example 3:] Player 2 (of the red team) kicks the ball ``upfield''. A blue-team robot kicks the ball a short distance, after which Player~2 kicks the ball again into the blue team's goal. This is a successful indirect kick, and the goal counts.
+  \item[Example 4:] Player 2 (of the red team), walks up to and dribbles the ball. To be an indirect kick, Player~2 must then stop, and visibly back away from the ball, before approaching to dribble a \textit{second} time. The robot then scores. This is a successful indirect kick.
 \end{description}
 
+\subsection{Field-side selection and initial Kick-off}
+\label{sec:field_side_and_initial_kickoff}
+
+    \begin{itemize}
+        \item The head referee throws a coin with one side for the home team and the other side for the away team according to the schedule. The team that wins the coin toss can either choose which goal to play for in the first half or take the kick-off.
+        \item Depending on the decision above, the opposing team is given either the kick-off or may choose which goal to play for in the first half.
```

Comment:
The winning seam can choose which side to play on or which team has kick off.
Subsequently, the second team can then decide if it wants to have kick off or can decide a side, but the team with the kick off selection has also the possibility to choose not to have kick off.